packages/mobile/src/harmony-native/components/Hint/Hint.tsx

```diff
@@ -21,12 +20,13 @@ export const Hint = (props: HintProps) => {
       backgroundColor='surface2'
       shadow='flat'
       border='strong'
+      // Width 100% is necessary to allow for text wrapping inside
+      // the flex container that wraps children
+      style={{ width: '100%' }}
       {...other}
     >
       <Icon size='l' color='default' />
-      <Text variant='body' color='default'>
-        {children}
-      </Text>
+      {children}
```

Comment:
Having arbitrary children in <Text> is a bad time.
This seems to be a better fix and confirmed that it works across android/ios and in both usages of Hint